### DISCUSSION TITLE
QDate Problem with date selection

### DISCUSSION THREAD

**Discussion by BekkerSmartseller** [quasarframework/quasar#17544] [General - Components / Directives / etc] [answered]
https://github.com/user-attachments/assets/0d533e29-efb4-4c61-913f-a59665eb0aae

How do I get rid of the rollover to another month when I select/cancel a date?
Date reset depends on the sequence of dates in the array. Perhaps there is a non-obvious option that cancels this behavior?

**Comment by TomWawIT** [comment]
Got the same problem here, looks like similar issue was fixed some time ago (https://github.com/quasarframework/quasar/discussions/11170) but it's back.

**Reply by rstoenescu** [reply]
Discovered the root cause in latest Quasar.
Fix will be available in Quasar v2.17.7

**Comment by rstoenescu** [comment, marked as answer]
Discovered the root cause in latest Quasar.
Fix will be available in Quasar v2.17.7
